src/components/Composer/implementation/index.native.tsx

```diff
@@ -158,6 +171,8 @@ function Composer(
                 props?.onBlur?.(e);
             }}
             onClear={onClear}
+            showSoftInputOnFocus={showSoftInputOnFocus}
+            contextMenuHidden={contextMenuHidden}
```

Comment:
Hi ✋ Coming from https://github.com/Expensify/App/issues/50682

It's expected that the context menu will show on Android when long presses it. So we think it has different behavior between Android and iOS, and we solve it with using `contextMenuHidden` on iOS only.